com.github.pyfa_org.Pyfa.json

```diff
@@ -0,0 +1,347 @@
+{
+  "id" : "com.github.pyfa_org.Pyfa",
+  "branch" : "stable",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "43",
```

Comment:
```suggestion
  "runtime-version" : "44",
```

Reply:
also you build gtk2, so why this runtime?

Reply:
Changed runtime to org.freedesktop.Platform//22.08